### ISSUE TITLE
orderingList: function remove is dangerous

### ISSUE THREAD

**Issue by jstefek** [richwidgets/richwidgets#111]
The function doesn't contain any context from which will be the items removed.
It simply calls `items.detach()`. So, in current version, this function can easily remove everything on page.
- open http://www.richwidgets.io/select/orderingList.html
- run `$('#list').orderingList('remove', $('*'));`
- expected: something like removing all list items
- have: everything on page is removed (`$('*')`)

Please, also update doc: this function accepts jQuery objects (not just any objects)

**Comment by bleathem** [comment]
Nice catch @jstefek!

**Comment by lfryc** [comment]
We could check for each item whether `$.contains($pluginRoot, item)`

**Comment by bleathem** [comment]
an alternate approach: `$pluginRoot.find(item).remove();`
